src/test/resources/net/masterthought/cucumber/location_bug.json

```diff
@@ -0,0 +1,114 @@
+[
+  {
+    "uri": "features/selenium.feature",
+    "id": "test",
+    "keyword": "Feature",
+    "name": "test",
+    "description": "",
+    "line": 1,
+    "elements": [
+      {
+        "id": "test;test1",
+        "keyword": "Scenario",
+        "name": "test1",
+        "description": "",
+        "line": 2,
+        "type": "scenario",
+        "steps": [
+          {
+            "keyword": "Given ",
```

Comment:
let's have two steps: one with new format, other with old one - this will eliminate noise of data that is not important in this test